### PR TITLE
Chris little patch editorial consistency

### PR DESCRIPTION
**Pull request by chris-little**
@tomkralidis Implements many OGC editorial consistency changes, but with half of a dozen issues (missing definitions or explanations to be done). See Issue #491

### CODE REVIEW

File: extensions/pubsub/standard/document.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= OGC API - Environmental Data Retrieval - Part 2: Publish-Subscribe workflow
+= OGC API - Environmental Data Retrieval - Part 2: Publish-Subscribe Workflow
```
Comment (tomkralidis) [inline extensions/pubsub/standard/document.adoc:1]: `Workflow` or `workflow` (sentence case exept for know terms)?

**Review by tomkralidis — CHANGES_REQUESTED** [review]
@chris-little very minor change request / comment for clarification, albeit in a critical part :)

File: extensions/pubsub/standard/document.adoc (hunk 1 above)
Comment (chris-little) [reply extensions/pubsub/standard/document.adoc:1]: But `Sentence case` not appropriate - it should be `Title Case` ! W not w. ;-)

And Carl says so.

**Comment by chris-little** [comment]
@tomkralidis Either way, I can merge now.

**Comment by chris-little** [comment]
As agreed at EDR API SWG 110

**Pull request merged**